src/annotator/test/guest-test.js

```diff
@@ -462,29 +462,20 @@ describe('Guest', () => {
       fakeSidebarFrame?.remove();
     });
 
-    it('hides sidebar when the user taps or clicks in the page', () => {
-      for (let event of ['click', 'touchstart']) {
+    it('hides sidebar on user "mousedown" or "touchstart" events in the document', () => {
+      for (let event of ['mousedown', 'touchstart']) {
         rootElement.dispatchEvent(new Event(event));
         assert.calledWith(guest.crossframe.call, 'closeSidebar');
+        guest.crossframe.call.resetHistory();
```

Comment:
Important to add this, otherwise it could succeed on the first loop and fail on the second and still pass the test.

Reply:
Good catch